wordpress/apache2-init.sh

```diff
@@ -5,25 +5,45 @@ set -eux
 URL=${WORDPRESS_DOMAIN:-localhost}
 OVERWRITE=${WORDPRESS_OVERWRITE:-"false"}
 REFRESH_ONLY=${WORDPRESS_REFRESH_ONLY:-"false"}
-
 # Defaults for optional fields:
 WORDPRESS_TITLE=${WORDPRESS_TITLE:-""}
 WORDPRESS_DESCRIPTION=${WORDPRESS_DESCRIPTION:-""}
+# bind variables
+WORDPRESS_THEME_LIST="${WORDPRESS_THEME_LIST:-""}"
+WORDPRESS_PLUGIN_LIST="${WORDPRESS_PLUGIN_LIST:-""}"
+
+# Install WP themes from ENV var WORDPRESS_THEME_LIST
+# $0
+function wp_theme_install () {
+    for theme in $WORDPRESS_THEME_LIST ; do
+        if wp theme is-installed "$theme" ; then  wp theme install "$theme" ; fi
```

Comment:
'if installed, install'? Looks like the condition needs to be inverted

Reply:
Blimey! Fixed.